azurerm/resource_arm_function_app.go

```diff
@@ -410,6 +441,14 @@ func flattenFunctionAppSiteConfig(input *web.SiteConfig) []interface{} {
 		result["always_on"] = *input.AlwaysOn
 	}
 
+	if input.Use32BitWorkerProcess != nil {
+		result["use_32_bit_worker_process"] = *input.Use32BitWorkerProcess
+	}
```

Comment:
could we add an acceptance test with the 64 bit worker process enabled? here's [an example of how we're doing that](https://github.com/terraform-providers/terraform-provider-azurerm/blob/master/azurerm/resource_arm_function_app_test.go#L14)

Reply:
so that we can get this merged, I've pushed an acceptance test for this - I hope you don't mind!

Reply:
not at all! thanks for pushing this, I got caught in higher priority tasks and couldn't focus on this now...